python/labbox_ephys/extractors/labboxephysrecordingextractor.py

```diff
@@ -222,9 +220,10 @@ def __init__(self, arg: Union[str, dict], download: bool=False):
             self._recording: se.RecordingExtractor = MdaRecordingExtractor(timeseries_path=data['raw'], samplerate=data['params']['samplerate'], geom=np.array(data['geom']), download=download)
         elif recording_format == 'nrs':
             self._recording: se.RecordingExtractor = NrsRecordingExtractor(**data)
-        # elif recording_format == 'nwb':
-        #     path0 = kp.load_file(data['path'])
-        #     self._recording: se.RecordingExtractor = NwbRecordingExtractor(path0, electrical_series_name='e-series')
+        elif recording_format == 'nwb':
+            from .nwbextractors import NwbRecordingExtractor
```

Comment:
Here's where we support the 'nwb' format. Note that we use a custom NwbRecordingExtractor because the spikeinterface one has a couple of issues.